pkg/scheduler/cache/util.go

```diff
@@ -29,11 +32,55 @@ func CreateNodeNameToInfoMap(pods []*v1.Pod, nodes []*v1.Node) map[string]*NodeI
 		}
 		nodeNameToInfo[nodeName].AddPod(pod)
 	}
+	imageStatesMap := createNodeImageStatesMap(nodes)
```

Comment:
I agree! Thanks Bobby, I learned better taste again :) 

(Also I named the new function as getXXX instead of createXXX because here the caller does not care create or get..)